documentation/openapi.yaml

```diff
@@ -137,6 +190,7 @@ paths:
       
   /logout:
     get:
+      summary: Invalidate API login session.
       responses:
         '200':
           description: OK
```

Comment:
you changed all "OK"s to a short description before. maybe we don't need the "summary" field because it is kind of the same. Or we use description as status-code descriptor to avoid the duplication of "summary" and "description".

Reply:
my point is: at leat in one way this should be consistent.

Reply:
"summary" is rendered in the HTML documentation, and describes the endpoint in general. I would prefer this field.